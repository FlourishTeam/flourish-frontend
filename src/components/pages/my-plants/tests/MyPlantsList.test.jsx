import React from 'react';
import { render, cleanup } from '@testing-library/react';
import MyPlantList from './MyPlantList';

describe('MyPlantList component', () => {
  afterEach(() => cleanup());
  it('renders MyPlantList', () => {
    const { asFragment } = render(<MyPlantList plants={[]} />);
    expect(asFragment()).toMatchSnapshot();
  });
});
