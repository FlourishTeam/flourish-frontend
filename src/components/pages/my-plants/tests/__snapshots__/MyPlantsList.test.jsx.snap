// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MyPlantList component renders MyPlantList 1`] = `
<DocumentFragment>
  <ul
    data-testid="plants"
  >
    <li>
      <a
        href="/myPlants/Amaryllis"
      >
        <div>
          <figure>
            <img
              alt="Amaryllis"
            />
            <figcaption>
              <p>
                Amaryllis
              </p>
            </figcaption>
          </figure>
        </div>
      </a>
    </li>
    <li>
      <a
        href="/myPlants/Pothos"
      >
        <div>
          <figure>
            <img
              alt="Pothos"
            />
            <figcaption>
              <p>
                Pothos
              </p>
            </figcaption>
          </figure>
        </div>
      </a>
    </li>
    <li>
      <a
        href="/myPlants/Congo Spear"
      >
        <div>
          <figure>
            <img
              alt="Congo Spear"
            />
            <figcaption>
              <p>
                Congo Spear
              </p>
            </figcaption>
          </figure>
        </div>
      </a>
    </li>
    <li>
      <a
        href="/myPlants/Canadian Ivy"
      >
        <div>
          <figure>
            <img
              alt="Canadian Ivy"
            />
            <figcaption>
              <p>
                Canadian Ivy
              </p>
            </figcaption>
          </figure>
        </div>
      </a>
    </li>
    <li>
      <a
        href="/myPlants/Rubber Plant"
      >
        <div>
          <figure>
            <img
              alt="Rubber Plant"
            />
            <figcaption>
              <p>
                Rubber Plant
              </p>
            </figcaption>
          </figure>
        </div>
      </a>
    </li>
    <li>
      <a
        href="/myPlants/Parlor Palm"
      >
        <div>
          <figure>
            <img
              alt="Parlor Palm"
            />
            <figcaption>
              <p>
                Parlor Palm
              </p>
            </figcaption>
          </figure>
        </div>
      </a>
    </li>
    <li>
      <a
        href="/myPlants/Chinese Evergreen"
      >
        <div>
          <figure>
            <img
              alt="Chinese Evergreen"
            />
            <figcaption>
              <p>
                Chinese Evergreen
              </p>
            </figcaption>
          </figure>
        </div>
      </a>
    </li>
    <li>
      <a
        href="/myPlants/Zebra Plant (Goeppertia sp.)"
      >
        <div>
          <figure>
            <img
              alt="Zebra Plant (Goeppertia sp.)"
            />
            <figcaption>
              <p>
                Zebra Plant (Goeppertia sp.)
              </p>
            </figcaption>
          </figure>
        </div>
      </a>
    </li>
    <li>
      <a
        href="/myPlants/Chinese Money Plant"
      >
        <div>
          <figure>
            <img
              alt="Chinese Money Plant"
            />
            <figcaption>
              <p>
                Chinese Money Plant
              </p>
            </figcaption>
          </figure>
        </div>
      </a>
    </li>
    <li>
      <a
        href="/myPlants/Heart Leaf Vine"
      >
        <div>
          <figure>
            <img
              alt="Heart Leaf Vine"
            />
            <figcaption>
              <p>
                Heart Leaf Vine
              </p>
            </figcaption>
          </figure>
        </div>
      </a>
    </li>
    <li>
      <a
        href="/myPlants/Bonnie Spider Plant"
      >
        <div>
          <figure>
            <img
              alt="Bonnie Spider Plant"
            />
            <figcaption>
              <p>
                Bonnie Spider Plant
              </p>
            </figcaption>
          </figure>
        </div>
      </a>
    </li>
    <li>
      <a
        href="/myPlants/Swiss Cheese Plant"
      >
        <div>
          <figure>
            <img
              alt="Swiss Cheese Plant"
            />
            <figcaption>
              <p>
                Swiss Cheese Plant
              </p>
            </figcaption>
          </figure>
        </div>
      </a>
    </li>
    <li>
      <a
        href="/myPlants/ZZ Plant"
      >
        <div>
          <figure>
            <img
              alt="ZZ Plant"
            />
            <figcaption>
              <p>
                ZZ Plant
              </p>
            </figcaption>
          </figure>
        </div>
      </a>
    </li>
    <li>
      <a
        href="/myPlants/Janet Craig Dracaena"
      >
        <div>
          <figure>
            <img
              alt="Janet Craig Dracaena"
            />
            <figcaption>
              <p>
                Janet Craig Dracaena
              </p>
            </figcaption>
          </figure>
        </div>
      </a>
    </li>
    <li>
      <a
        href="/myPlants/Corn Plant"
      >
        <div>
          <figure>
            <img
              alt="Corn Plant"
            />
            <figcaption>
              <p>
                Corn Plant
              </p>
            </figcaption>
          </figure>
        </div>
      </a>
    </li>
    <li>
      <a
        href="/myPlants/Aluminum Plant"
      >
        <div>
          <figure>
            <img
              alt="Aluminum Plant"
            />
            <figcaption>
              <p>
                Aluminum Plant
              </p>
            </figcaption>
          </figure>
        </div>
      </a>
    </li>
    <li>
      <a
        href="/myPlants/Areca Palm"
      >
        <div>
          <figure>
            <img
              alt="Areca Palm"
            />
            <figcaption>
              <p>
                Areca Palm
              </p>
            </figcaption>
          </figure>
        </div>
      </a>
    </li>
    <li>
      <a
        href="/myPlants/Arrowhead Plant"
      >
        <div>
          <figure>
            <img
              alt="Arrowhead Plant"
            />
            <figcaption>
              <p>
                Arrowhead Plant
              </p>
            </figcaption>
          </figure>
        </div>
      </a>
    </li>
    <li>
      <a
        href="/myPlants/Asparagus Fern"
      >
        <div>
          <figure>
            <img
              alt="Asparagus Fern"
            />
            <figcaption>
              <p>
                Asparagus Fern
              </p>
            </figcaption>
          </figure>
        </div>
      </a>
    </li>
    <li>
      <a
        href="/myPlants/Baby Rubber Plant"
      >
        <div>
          <figure>
            <img
              alt="Baby Rubber Plant"
            />
            <figcaption>
              <p>
                Baby Rubber Plant
              </p>
            </figcaption>
          </figure>
        </div>
      </a>
    </li>
    <li>
      <a
        href="/myPlants/Baby's Tears"
      >
        <div>
          <figure>
            <img
              alt="Baby's Tears"
            />
            <figcaption>
              <p>
                Baby's Tears
              </p>
            </figcaption>
          </figure>
        </div>
      </a>
    </li>
    <li>
      <a
        href="/myPlants/Bird's Nest Fern"
      >
        <div>
          <figure>
            <img
              alt="Bird's Nest Fern"
            />
            <figcaption>
              <p>
                Bird's Nest Fern
              </p>
            </figcaption>
          </figure>
        </div>
      </a>
    </li>
    <li>
      <a
        href="/myPlants/Boston Fern"
      >
        <div>
          <figure>
            <img
              alt="Boston Fern"
            />
            <figcaption>
              <p>
                Boston Fern
              </p>
            </figcaption>
          </figure>
        </div>
      </a>
    </li>
    <li>
      <a
        href="/myPlants/Angel Wings"
      >
        <div>
          <figure>
            <img
              alt="Angel Wings"
            />
            <figcaption>
              <p>
                Angel Wings
              </p>
            </figcaption>
          </figure>
        </div>
      </a>
    </li>
    <li>
      <a
        href="/myPlants/Orbifolia Prayer Plant"
      >
        <div>
          <figure>
            <img
              alt="Orbifolia Prayer Plant"
            />
            <figcaption>
              <p>
                Orbifolia Prayer Plant
              </p>
            </figcaption>
          </figure>
        </div>
      </a>
    </li>
    <li>
      <a
        href="/myPlants/Rattlesnake Plant"
      >
        <div>
          <figure>
            <img
              alt="Rattlesnake Plant"
            />
            <figcaption>
              <p>
                Rattlesnake Plant
              </p>
            </figcaption>
          </figure>
        </div>
      </a>
    </li>
    <li>
      <a
        href="/myPlants/Cape Sundew"
      >
        <div>
          <figure>
            <img
              alt="Cape Sundew"
            />
            <figcaption>
              <p>
                Cape Sundew
              </p>
            </figcaption>
          </figure>
        </div>
      </a>
    </li>
    <li>
      <a
        href="/myPlants/Cardboard Palm"
      >
        <div>
          <figure>
            <img
              alt="Cardboard Palm"
            />
            <figcaption>
              <p>
                Cardboard Palm
              </p>
            </figcaption>
          </figure>
        </div>
      </a>
    </li>
    <li>
      <a
        href="/myPlants/Watch Chain Plant"
      >
        <div>
          <figure>
            <img
              alt="Watch Chain Plant"
            />
            <figcaption>
              <p>
                Watch Chain Plant
              </p>
            </figcaption>
          </figure>
        </div>
      </a>
    </li>
    <li>
      <a
        href="/myPlants/Creeping Fig"
      >
        <div>
          <figure>
            <img
              alt="Creeping Fig"
            />
            <figcaption>
              <p>
                Creeping Fig
              </p>
            </figcaption>
          </figure>
        </div>
      </a>
    </li>
    <li>
      <a
        href="/myPlants/Elephant Ear"
      >
        <div>
          <figure>
            <img
              alt="Elephant Ear"
            />
            <figcaption>
              <p>
                Elephant Ear
              </p>
            </figcaption>
          </figure>
        </div>
      </a>
    </li>
    <li>
      <a
        href="/myPlants/English Ivy"
      >
        <div>
          <figure>
            <img
              alt="English Ivy"
            />
            <figcaption>
              <p>
                English Ivy
              </p>
            </figcaption>
          </figure>
        </div>
      </a>
    </li>
    <li>
      <a
        href="/myPlants/Fiddle Leaf Fig"
      >
        <div>
          <figure>
            <img
              alt="Fiddle Leaf Fig"
            />
            <figcaption>
              <p>
                Fiddle Leaf Fig
              </p>
            </figcaption>
          </figure>
        </div>
      </a>
    </li>
    <li>
      <a
        href="/myPlants/Gold Dust Dracaena"
      >
        <div>
          <figure>
            <img
              alt="Gold Dust Dracaena"
            />
            <figcaption>
              <p>
                Gold Dust Dracaena
              </p>
            </figcaption>
          </figure>
        </div>
      </a>
    </li>
  </ul>
</DocumentFragment>
`;
